SETTINGS.rst

```diff
@@ -0,0 +1,122 @@
+=====================
+ Basic configuration
+=====================
+
+First, locate your `settings.py`. Since this is a regular python file,
+experienced users can also write more advanced configurations with e.g. swithing
+between two sets of configs. This also means, that the syntax need to be correct
+for OpenSlides to start.
+
+All presented settings must be written `<SETTINGS_NAME> = <value>` to follow the
+correct syntax.
+
+The `settings.py` is just an extension for Django settings. Please visit the
+`Django settings documentation
+<https://docs.djangoproject.com/en/2.2/ref/settings/>`_ to get an overview about
+all existing settings.
+
+SECURITY
+========
+
+For `DEBUG` and `SECRET_KEY` see the sections in the django settings
+documenataion.
+
+`RESET_PASSWORD_VERBOSE_ERRORS`: Default: `True`. Set to `False` to disable.
+Controls the verbosity on errors during a reset password. If enabled, an error
+will be shown, if there does not exist a user with a given email address. So one
+can check, if a email is registerd. If this is not wanted, disable verbose
+messages. An success message will always be shown.
+
+`AUTH_PASSWORD_VALIDATORS`: Add custom password validators, e.g. a min-length
+validator. See `django auth docs
+<https://docs.djangoproject.com/en/2.2/topics/auth/passwords/#module-django.contrib.auth.password_validation>`_
+for mor information.
+
+Directories
+===========
+
+`OPENSLIDES_USER_DATA_DIR`: The path, where all user data is saved, like static
+files, mediafiles and the default database. This path can be different to the
+location of the `settings.py`.
+
+`STATICFILES_DIRS` and `STATIC_ROOT`: Managing static files. Because the clint
+is not delivered by the serve anymore, these settings are obsolete.
+
+`MEDIA_ROOT`: The location of mediafiles. The default is a `media` folder insile
+`OPENSLIDES_USER_DATA_DIR`, but can be altered to another path.
+
+Email
+=====
+
+Please refer to the Django settings documentation. A changed email backend is
+usefull for debugging to print all email the the console::
+
+   EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
```

Comment:
here we should add our settings option: 
```DEFAULT_FROM_EMAIL = 'noreply@example.com'```

Reply:
This is not our own setting, but one of djangos pre-filled: https://docs.djangoproject.com/en/2.2/ref/settings/#default-from-email

Reply:
Right.